controllers/server.go

```diff
@@ -94,6 +95,7 @@ func getStatus(w http.ResponseWriter, r *http.Request) {
 		DB:           database.IsConnected(),
 		Broker:       mq.IsConnected(),
 		LicenseError: licenseErr,
+		IsEE:         servercfg.IsPro,
```

Comment:
Going forward, the plan is to use only `Pro` terminology instead of `EE`

Reply:
changed